ironfish/src/rpc/clients/ipcClient.ts

```diff
@@ -24,10 +24,11 @@ export class RpcIpcClient extends RpcSocketClient {
 
   constructor(
     connection: Partial<RpcClientConnectionInfo> = {},
+    authToken: string,
```

Comment:
I don't think we want authentication in the IPC protocol right?

Reply:
removed